src/main/java/net/kodehawa/mantarobot/utils/Utils.java

```diff
@@ -509,7 +501,15 @@ public static boolean handleIncreasingRatelimit(IncreasingRateLimiter rateLimite
             try {
                 //noinspection ResultOfMethodCallIgnored
                 Long.parseUnsignedLong(u);
-                User user = MantaroBot.getInstance().getShardManager().getUserById(u);
+                User user;
+
+                try {
+                    user = MantaroBot.getInstance().getShardManager().retrieveUserById(u).complete();
```

Comment:
there's an event argument, might be worth fetching member instead of user

Reply:
Fixed